src/neural/cuda/winograd_helper.inc

```diff
@@ -68,6 +68,8 @@ __device__ __forceinline__ float activate(float cVal,
     case ACTIVATION_SWISH:
       cVal /= (1.0f + __expf(-cVal));
       break;
+    default:
```

Comment:
I'd suggest to explicitly list all the enums, so that when a new enum is added, it's flagged with compile-time warning (or better error).
Also, shouldn't it be some kind of exception rather that ignoring this silently?

Reply:
Apparently cuda has a `__trap()` function to trigger an illegal instruction abort. Does that sound OK?

Reply:
Ah it's a CUDA code.. Yeah, sounds good!